package.json

```diff
@@ -69,7 +69,7 @@
     "parse-path": "^5.0.0",
     "parse-url": "8.1.0",
     "terser": "^5.14.2",
-    "vm2": "^3.9.17",
```

Comment:
was this intentional?

Reply:
Security audit update.

Reply:
https://bitgo.slack.com/archives/C010AEXLLCR/p1684247091685949?thread_ts=1684246605.689579&cid=C010AEXLLCR

Reply:
@mohammadalfaiyazbitgo shouldn't there be a package-lock change as well with this? Did you run `yarn`?

Reply:
yeah forgot that, I'll update